src/main/java/de/retest/web/DefaultValuesProvider.java

```diff
@@ -73,20 +83,13 @@ public Serializable getDefaultValue( final IdentifyingAttributes comp, final Str
 	}
 
 	public boolean isDefault( final String tag, final String attribute, final String attributeValue ) {
-		final String defaultValue = getDefaultValue( tag, attribute );
 		if ( attributeValue == null || attributeValue.trim().isEmpty() ) {
 			return true;
 		}
-		if ( attributeValue.equalsIgnoreCase( defaultValue ) ) {
-			return true;
-		}
-		if ( attributeValue.equals( "auto" ) ) {
-			return true;
-		}
-		if ( attributeValue.equals( "normal" ) ) {
+		if ( attributeValue.equalsIgnoreCase( getDefaultValue( tag, attribute ) ) ) {
 			return true;
 		}
-		if ( attributeValue.equals( "0px" ) ) {
+		if ( commonDefaults.contains( attributeValue ) ) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this if-then-else statement by a single return statement. ([squid:S1126](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1126))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=67&issues=AWX2C_fz2IDoRcZ5MkX0&open=AWX2C_fz2IDoRcZ5MkX0)

[ISSUE_KEY]: AWX2C_fz2IDoRcZ5MkX0